Ubiquity Journals.js

```diff
@@ -0,0 +1,156 @@
+			if(!items) return true;
+			var urls = [];
+			for(var i in items) {
+				urls.push(i);
+			}
+			ZU.processDocuments(urls, scrape);
+		})
+	} else {
+		scrape(doc, url);
+	}
+}
+
+function getSearchResults(doc, checkOnly) {
+	var results = ZU.xpath(doc,
+			'//div[@class="article-caption"]/div[@class="caption-text"]/a[contains(@href, "/articles/10.")]'
+		),
+		items = {},
+		found = false;
+	for(var i = 0; i < results.length; i++) {
+		var title = results[i].textContent;
+		if(!title) continue;
+		if(checkOnly) return true;
+		found = true;
+		title = title.trim();
+		items[results[i].href] = title;
+	}
+	return found ? items : false;
+}
+
+function scrape(doc, url) {
+	var abstract = ZU.xpathText(doc, '//meta[@name="DC.Description"]/@content');
+	var translator = Zotero.loadTranslator('web');
+	// use the Embedded Metadata translator
+	translator.setTranslator('951c027d-74ac-47d4-a107-9c3069ab7b48');
+	translator.setDocument(doc);
+	translator.setHandler('itemDone', function(obj, item) {
+		if(abstract) item.abstractNote = abstract.trim();
+		item.complete();
+	});
+	translator.getTranslatorObject(function(trans) {
+		trans.doWeb(doc, url);
+	});
+}/** BEGIN TEST CASES **/
+var testCases = [
+	{
+		"type": "web",
+		"url": "http://www.ancient-asia-journal.com/articles/",
+		"items": "multiple"
+	},
+	{
+		"type": "web",
+		"url": "http://journal.eahn.org/articles/10.5334/ah.bd/",
+		"items": [
+			{
+				"itemType": "journalArticle",
+				"title": "Handymen, Hippies and Healing: Social Transformation through the DIY Movement (1940s to 1970s) in North America",
+				"creators": [
+					{
+						"firstName": "Cathy",
+						"lastName": "Smith",
+						"creatorType": "author"
+					}
+				],
+				"date": "2014-03-19T12:43:17",
+				"DOI": "10.5334/ah.bd",
+				"ISSN": "2050-5833",
+				"abstractNote": "<p class=\"p1\">This paper explores the relation between the ‘DIY’ (‘do-it-yourself’) movement and ‘DIY architecture’, and the notion of social transformation, in examples of DIY manuals and discourse of North America drawn from the 1940s to the 1970s. The DIY movement emerged as a significant phenomenon in North America of the 1950s, where it was associated with a mainstream audience and a residential market. By the 1960s, the DIY approach was embraced by the North American counterculture as a self-sustaining sensibility that could overcome a reliance on the mainstream, consumerist society that spurned it. On the surface, the association of DIY with the counterculture and countercultural architects appears to denote a significant ideological shift from its original association with the beliefs and culture of mainstream North America and the nuclear family. However, one of the key characterisations of the DIY movement identified in the present paper is the way it is bound to the notion of social identity and transformation, regardless of ideology. Particular attention is paid to DIY manuals and discourse of the 1950s.</p>",
```

Comment:
I don't think `cleanTags` will work with escaped html tags, cf. https://github.com/zotero/zotero/blob/master/chrome/content/zotero/xpcom/utilities.js#L265 .  However, it should be easy to come up with something similiar, maybe only this case with the `<p>`-Tag. E.g. something like
```
item.abstractNote = abstract.replace(/&lt;p[^&>]*&gt;/g, "").trim();
```